projects/demo-integrations/cypress/support/helpers/hover.ts

```diff
@@ -0,0 +1,10 @@
+export const tuiHover = (
+    $subject: Cypress.PrevSubjectMap<void>[Cypress.PrevSubject],
+): Cypress.Chainable<unknown> =>
+    cy
+        .wrap($subject, {log: false})
+        .trigger(`mouseover`, {force: true})
+        .trigger(`mouseenter`, {force: true})
+        .invoke(`show`)
+        .click({force: true}) // emulate hover
```

Comment:
sorry, I missed it, why tuiHover needs a click? 
e2e for [hint](https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo-integrations/cypress/tests/desktop/core/hint/hint.cy.ts) (for example) works without click 🤔

Reply:
fixed